src/objectTranslations/DataImport__c-es.objectTranslation

```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <plural>false</plural>
+        <value><!-- NPSP Data Import --></value>
+    </caseValues>
+    <caseValues>
+        <plural>true</plural>
+        <value><!-- NPSP Data Import --></value>
+    </caseValues>
+    <fields>
+        <label>Cuenta1 Estado de importación</label>
+        <name>Account1ImportStatus__c</name>
+    </fields>
+    <fields>
+        <label>Cuenta1 Importada</label>
+        <name>Account1Imported__c</name>
+        <relationshipLabel>NPSP Data Imports (Cuenta1 Importada)</relationshipLabel>
+    </fields>
+    <fields>
+        <help>Cuenta1.BillingCity</help>
```

Comment:
these translations of the help text will break the NPSP data importer.  the help text is used to specify which object and field the import should map to.  So Cuenta1 needs to stay as Account1.  the easiest fix is to just strip out all help text entries for this translation file.